tictactoe/tic.js

```diff
@@ -1 +1,63 @@
-console.log('Hello world!!')
\ No newline at end of file
+let model = ['','','','','','','','','']
```

Comment:
Let us refactor using a 2D array for model. It will help when you do AI (Computer play) later. 
`let model = [['','',''],  ['','',''], ['','','']]
`

Reply:
no, it doesnt